src/main/scala/com/sbuslab/sbus/rabbitmq/RabbitMqTransport.scala

```diff
@@ -34,8 +34,11 @@ class RabbitMqTransport(conf: Config, authProvider: AuthProvider, actorSystem: A
 
   private val log = Logger(LoggerFactory.getLogger("sbus.rabbitmq"))
 
+  private val recordSBusInteractions =
+    System.getProperties.containsKey("record.sbus")
```

Comment:
@kulikov The reason why the typesafe config is not used, is so that this property can be used in the LogBack.xml to configure the additional appender.  To use the TypeSafe config you need to have already have started the logger.  To utilise this feature the developer would add -Drecord.sbus to the gradle CLI. e.g

`gradle :exchange:secured-loan-service:run -Drecord.sbus`